server/pages/index.js

```diff
@@ -498,6 +558,37 @@ module.exports = function() {
 			}
 		} );
 
+	// This used to log to tracks Content Security Policy violation reports sent by browsers
```

Comment:
Grammar note only for clarity here

"This used to" reads "At some point in the past we were using this function to log to Tracks"
"This is used to" read "We are using this function to log to Tracks"

Small word omission but a heavy impact on meaning.